ui/components-react/src/components-react/filter-builder/FilterBuilderRule.tsx

```diff
@@ -31,7 +34,10 @@ export interface PropertyFilterBuilderRuleRendererProps {
   path: string[];
   /** Rule to render. */
   rule: PropertyFilterBuilderRule;
-  isRemovable?: boolean;
+  /** Function to add rule to group */
+  onAddRule: () => void;
```

Comment:
Why does it need `onAddRule` callback in props? It has access to `PropertyFilterBuilderActions` and uses them for rule removal.

Reply:
it at least needs the onNewRuleAdded callback from FilterBuilderRuleGroup since that is called after actions.addItem to focus the newly added row

Reply:
In that case I would prefer renaming this to `onRuleAdded` and pass `onNewRuleAdded` from `FilterBuilderRuleGroup` to this component.